core/runtime/jsruntime.rs

```diff
@@ -869,7 +869,15 @@ impl JsRuntime {
       );
     }
 
-    context.set_slot(scope, context_state.clone());
+    // SAFETY: We need to initialize the slot. rusty_v8 currently segfaults
+    // when call `clear_all_slots`.
+    unsafe {
+      context.set_slot(scope, ());
```

Comment:
Can we address this in rusty_v8 before landing this PR?

Reply:
https://github.com/denoland/rusty_v8/pull/1497

Reply:
Fixed